packages/shared/package.json

```diff
@@ -15,6 +15,7 @@
 		"build": "tsc -p tsconfig.build.json && tsc-alias -p tsconfig.build.json"
 	},
 	"devDependencies": {
-		"zod": "3.23.8"
+		"zod": "3.23.8",
+		"date-fns": "3.6.0"
```

Comment:
Move to dependencies. As you do that, move zod as well to dependencies, please

Reply:
Check it, please